content/04.network_applications.md

```diff
@@ -40,18 +40,44 @@ In the future, it may be beneficial for these models is to incorporate other typ
 We put particular emphasis on an emerging set of techniques: those that project knowledge graphs into a low dimensional space.
 
-#### Disease and Gene Interactions
-
-1. Mention disease gene prioritization
-2. Mention Disease gene associations
-
-#### Protein Protein Interactions
-
-1. Mention predicting genes interacting genes
+![
+Overview of biomedical applications that make use of knowledge graphs.
+Categories consist of: (a) Multi-Omic applications, (b) Pharmaceutical Applications and (c) Clinical Applications.
+](images/figures/unifying_applications_overview.png){#fig:unifying_applications}
+
+#### Multi-Omic Applications
+
+Multi-omic applications consist of studying genes (genomics), RNA (transcriptomics) and proteins (proteomics).
+Approaches in this category use knowledge graphs to establish connections between biological entities (genes, RNA, proteins) themselves and diseases via tasks such as gene prioritization [@doi:10.1093/jamia/ocy117], predicting interactions between proteins [@doi:10.1089/cmb.2012.0273;10.1186/1752-0509-9-S1-S9], and detecting MiRNA-disease associations [@doi:10.1155/2017/2498957].
+In this section we discuss on multi-omic applications that utilize projecting knowledge graphs into a low dimensional space to make connections.
+
+Knowledge graphs have been used as recommendation systems to establish links between RNA with disease and proteins with other proteins.
+Shen et al. [@doi:10.1155/2017/2498957] used an algorithm called collaborative filtering to establish an association between MiRNA and diseases.
+The authors constructed an MiRNA-Disease network using the HMDDD database [@doi:10.1093/nar/gky1010] and generated an adjacency matrix with the rows representing MiRNA and the columns representing diseases.
+This adjacency matrix was decomposed into small rectangular matrices using SVD, then these matricies were used to calculate similarity scores between the MiRNA and diseases.
+High scores implicated a high likelihood that a given MiRNA has an association with a given disease [@doi:10.1155/2017/2498957].
+Other approaches have built off of Shen et al.'s [@doi:10.1155/2017/2498957] work by incorpoating novel ways to perform matrix factorization [@doi:10.3390/genes10020080; @doi:10.1186/s12859-019-2956-5;@doi:10.1186/s12859-019-3260-0] or by integrating machine learning models in conjunction with matrix factorization [@doi:10.1109/TCBB.2019.2937774].
+These methods provided high AUROCs, but new discoveries were hard to validate as there is limited experimental data [@doi:10.1155/2017/2498957].
+Apart from MiRNA, collaborative filtering has been used to predict protein-protein interactions [@doi:10.1109/BIBM.2010.5706537; @doi:10.1089/cmb.2012.0273; @doi:10.1186/1752-0509-9-S1-S9].
+Future work could include doing a blinded literature search to increase validation power or incorporate other techniques as deep learning to increase detection power.
```

Comment:
```suggestion
Though extensive validation of newly generated candidates may be impractical, it would be helpful to see future efforts in this space include a blinded literature search for prioritized and randomly selected candidates as part of the standard evaluation of such approaches.
```